djstripe/urls.py

```diff
@@ -1,5 +1,7 @@
 # -*- coding: utf-8 -*-
```

Comment:
full module docstring please

author is just pydanny
https://github.com/kavdev/dj-stripe/blame/%23162-api-updates-through_2015-07-28/djstripe/urls.py